tests/RelationsTest.php

```diff
@@ -470,6 +470,25 @@ public function testMorph(): void
         $relations = $photos[1]->getRelations();
         $this->assertArrayHasKey('hasImage', $relations);
         $this->assertInstanceOf(Client::class, $photos[1]->hasImage);
+
+        // inverse
+        $photo = Photo::query()->create(['url' => 'https://graph.facebook.com/hans.thomas/picture']);
+        $client = Client::create(['name' => 'Hans Thomas']);
```

Comment:
@hans-thomas: Was there a particular reason you used `Photo::query()->create()` above but `Client::create()` here? Best I could tell, they resolve to the same `Builder::create()` method.

`Client::create()` just ends up forwarding the call through `Model::__call()`.